x/committee/types/proposal.go

```diff
@@ -110,30 +69,29 @@ func (ccp CommitteeChangeProposal) ProposalRoute() string { return RouterKey }
 // ProposalType returns the type of the proposal.
 func (ccp CommitteeChangeProposal) ProposalType() string { return ProposalTypeCommitteeChange }
 
+func (ccp CommitteeChangeProposal) GetNewCommittee() Committee {
+	committee, err := UnpackCommittee(ccp.NewCommittee)
+	if err != nil {
+		panic(err)
+	}
+	return committee
+}
```

Comment:
If I'm understanding correctly, the `CommitteeChangeProposal` will need a `UnpackInterfaces` method as well. Otherwise the cached value won't be populated when unmarshalled.

Reply:
maybe add a `var _ UnpackInterfacesMessage = CommitteeChangeProposal{}` type check?

Reply:
yep, that is correct. i'll add this in